### PR TITLE
📚 docs: deploying a new instance

### PR DESCRIPTION
**Pull request by thisislawatts**
Documentation around Firebase configuration required to deploy new instance

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 FAST_REFRESH=false
 SITE_NAME=Project Kamp
+REACT_APP_SITE_VARIANT=dev_site
```
Comment (thisislawatts) [inline .env:3]: Locally I have switched to using the `.env` file to drive which Firebase Config to use

File: src/config/config.ts [hunk 2]
```diff
@@ -47,71 +47,6 @@ const branch = e.REACT_APP_BRANCH as string
 // On dev sites user can override default role
 const devSiteRole: UserRole = localStorage.getItem('devSiteRole') as UserRole
```
Comment (thisislawatts) [inline src/config/config.ts:49]: This can really be ignored, there is some noise here from unrelated workflow changes I have been experimenting with locally.

File: src/pages/PageList.tsx [hunk 3]
```diff
@@ -161,12 +167,20 @@ const termsPolicy = {
...
 }
 
+export async function getAvailablePageList(): Promise<IPageMeta[]> {
+  const enabledModules: string[] = await getSupportedModules()
+
+  return COMMUNITY_PAGES.filter(pageItem =>
+    enabledModules.includes(pageItem.moduleName || ''),
+  )
+}
+
```
Comment (thisislawatts) [inline src/pages/PageList.tsx:177]: At the moment we are only looking to filter `COMMUNITY_PAGES` based on module configuration.

**Review by davehakkens — APPROVED** [review]
I approve but stop there because "This PR is unlikely to be ever be merged"

**Comment by thisislawatts** [comment]
@davehakkens, updated the PR description. Originally I had put a lot of hacky code changes into this branch but those have all been removed whilst the functionality was being merged into the production instance.
